tests/lts_compatibility.py

```diff
@@ -236,6 +237,18 @@ def run_code_upgrade_from(
                 network.retire_node(primary, node)
                 if primary == node:
                     primary, _ = network.wait_for_new_primary(primary)
+                    spec = inspect.signature(network.txs.issue)
+                    if "record_claim" in spec.parameters:
+                        LOG.info("Run transaction with additional claim")
+                        txid = network.txs.issue(
+                            network, number_txs=1, record_claim=True
+                        )
```

Comment:
As discussed, I don't think the JS app has been upgraded at this point (still using 1.x) so I believe the claim will not be recorded.

Reply:
Absolutely, this and several more issues fixed as discussed